README.md

```diff
@@ -102,7 +102,11 @@ To run Zilliqa, we recommend the following minimum system requirements:
     pkill zilliqa
     ``` 
 
-## Joining public testnet 
+## GPU Mining
```

Comment:
To keep README.md simple and clean, we perhaps don't need this section for now as there are little information to offer here and the link to wiki has been added to **Available features**.

See https://github.com/Zilliqa/Zilliqa/blame/master/README.md#L44